esphome/components/display/display_buffer.cpp

```diff
@@ -591,6 +591,18 @@ void Animation::prev_frame() {
   }
 }
 
+void Animation::set_frame(int frame) {
+  unsigned abs_frame = abs(frame);
+
+  if (abs_frame < this->animation_frame_count_) {
+    if (frame > 0) {
```

Comment:
Shouldnt you be able to set to frame 0 too?

```suggestion
    if (frame >= 0) {
```

Reply:
D'oh, yeah.

Reply:
You most definitely should. Thanks! Not sure how I missed that :)